infra/ansible/playbooks/setup.yaml

```diff
@@ -11,44 +11,53 @@
           - git
           - ufw
           - make
+          - gcc
         state: latest
         update_cache: true
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
 
     # Allow SSH connections and deny by default
     - name: UFW - Allow SSH connections
       become: true
       community.general.ufw:
         rule: allow
         name: OpenSSH
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
 
     - name: UFW - Enable and deny by default
       become: true
       community.general.ufw:
         state: enabled
         default: deny
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
 
     # Create basic directories
     - name: Create basic directories if do not exist
-      become: true
       ansible.builtin.file:
-        path: /home/{{ app_user }}/{{ item }}
+        path: /home/{{ ansible_user }}/{{ item }}
         state: directory
         mode: '0755'
-        owner: '{{ app_user }}'
-        group: '{{ app_user }}'
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
       loop:
         - repos
         - config
         - services
+        - .keystores
+
+    - name: Enable linger for {{ ansible_user }}
+      shell: sudo loginctl enable-linger {{ ansible_user }}
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
 
-    - name: Enable linger for {{ app_user }}
-      become: true
-      shell: loginctl enable-linger {{ app_user }}
 
     # Clone Aligned repository
```

Comment:
let's remove this as we are cloning in each component's playbook, and also because it's commented out